compute_endpoint/globus_compute_endpoint/self_diagnostic.py

```diff
@@ -19,26 +18,31 @@
 from globus_compute_sdk.sdk.web_client import WebClient
 
 
-def cat(path: str, wildcard: bool = False, max_bytes: int | None = None):
-    def kernel() -> None:
-        full_path = os.path.expanduser(path)
+def cat(path: str, wildcard: bool = False, max_bytes: int = 0):
+    max_bytes = max(0, max_bytes)
```

Comment:
This line appears to exclude negative numbers without fanfare. I recommend testing negative numbers in the parametrized "respect max_bytes" test.

Reply:
Och; good call.  I waffled on the implementation here multiple times when I was in there; totally should have added negatives in there.  Thanks!

Reply:
And addressed.  :muscle: